ouroboros-consensus-diffusion/test/consensus-test/Test/Consensus/Genesis/Tests/DensityDisconnect.hs

```diff
@@ -278,3 +292,78 @@ prop_densityDisconnectMonotonic =
     gen = do
       gt <- genChains (QC.choose (1, 4))
       evolveBranches (initCandidates gt)
+
+prop_densityDisconnectChainSel :: Property
```

Comment:
We have been asked in earlier reviews to document what this properties are testing.